esmvaltool/recipes/recipe_climwip.yml

```
@@ -0,0 +1,122 @@
+# ESMValTool
+# recipe_climwip.yml
+---
+documentation:
+  description: EUCP ClimWIP
+
+  authors:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+    - camphuijsen_jaro
+
+  maintainer:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+
+  references:
+    - brunner2019
+    - lorenz2018
+    - knutti2017
+
+datasets: &model_data
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: CanESM2, project: CMIP5, exp: [historical, rcp45], ensemble: "r(1:5)i1p1"}
+
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp45], ensemble: "r(1:4)i1p1"}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp60], ensemble: "r(1:4)i1p1"}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: "r(1:4)i1p1"}
+
+  - {dataset: CSIRO-Mk3-6-0, project: CMIP5, exp: [historical, rcp26], ensemble: "r(1:10)i1p1"}
+
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp26], ensemble: r1i1p1}
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+    extract_region:
+      start_longitude: -10.0
```

```suggestion
      start_longitude: 0.0
```

Regions including the zero meridian currently do not work for files which include bounds. For now use a slightly different region. 
https://github.com/ESMValGroup/ESMValCore/issues/799

---

I tested the fix (https://github.com/ESMValGroup/ESMValCore/tree/kluge_fix_issue799),  and think it only fixes part of the issue, crossing the 0 meridian seems to be a problem. So I would definitely avoid it at this point.